core/server/worker/src/main/java/alluxio/worker/block/meta/UnderFileSystemBlockMeta.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.block.meta;
+
+import alluxio.thrift.LockBlockTOptions;
+
+/**
+ * This class represents the metadata of a block that is in UFS.
+ */
+public final class UnderFileSystemBlockMeta {
+  private final ConstMeta mConstMeta;
+
+  /** The set of session IDs to be committed. */
+  private boolean mCommitPending;
+
+  /**
+   * The constant metadata of this UFS block.
+   */
+  public static final class ConstMeta {
+    public final long mSessionId;
```

Comment:
why session id is part of `ConstMeta`, is the session permanent and never changed?